packages/docs-page/render-page-mdx.js

```diff
@@ -1,22 +1,26 @@
 import path from 'path'
 import { serialize } from 'next-mdx-remote/serialize'
 import markdownDefaults from '@hashicorp/platform-markdown-utils'
+import { anchorLinks } from '@hashicorp/remark-plugins'
 import grayMatter from 'gray-matter'
 
 async function renderPageMdx(
   mdxFileString,
   { mdxContentHook = (c) => c, remarkPlugins = [], scope } = {}
 ) {
+  const headings = []
+
   const { data: frontMatter, content: rawContent } = grayMatter(mdxFileString)
   const content = mdxContentHook(rawContent)
   const mdxSource = await serialize(content, {
     mdxOptions: markdownDefaults({
       resolveIncludes: path.join(process.cwd(), 'content/partials'),
-      addRemarkPlugins: remarkPlugins,
+      addRemarkPlugins: [...remarkPlugins, [anchorLinks, { headings }]],
```

Comment:
`anchorLinks` is already present within the markdown defaults, so this I think will double the plugin. Instead, the headings option can be passed in via options:

https://github.com/hashicorp/remark-plugins/blob/main/index.js#L16
https://github.com/hashicorp/web-platform-packages/blob/main/packages/markdown-utils/index.ts#L29
https://github.com/hashicorp/web-platform-packages/blob/main/packages/markdown-utils/index.ts#L15

Reply:
Fixed! No more doubling of `anchorLinks` plugin